### PR TITLE
Create adamski.json

### PR DESCRIPTION
**Pull request by edtimer**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

**Review by Stef-DP — CHANGES_REQUESTED** [review]
There is no records

File: adamski.json
Comment (Stef-00012) [inline adamski.json:1]: also the file should be in the domains folder

**Review by Stef-DP — DISMISSED** [review]

File: adamski.json
Comment (edtimer) [reply adamski.json:1]: Sorry for the oversight. fixed

File: domains/adamski.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "owner": {
+    "username": "edtimer",
+    "email": "b4timer@protonamil.com"
+  },
+  "record": {
+    "CNAME": "adamski.pages.dev"
```
Comment (CuteDog5695) [inline domains/adamski.json:7]: Can you add more stuff to your website?

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/adamski.json (hunk 1 above)
Comment (edtimer) [reply domains/adamski.json:7]: Oh, alright noted. Thought I can take my time to design it better.

Comment (CuteDog5695) [reply domains/adamski.json:7]: You can add like an about me to your website for example

Comment (edtimer) [reply domains/adamski.json:7]: @CuteDog5695  Thank you. I have uploaded more to the site. Regards.

**Comment by edtimer** [comment]
image





done

**Review by iostpa — APPROVED** [review]
Seems good to me now

**Pull request merged**